packages/next-mdx/readme.md

```diff
@@ -76,3 +78,74 @@ module.exports = withMDX({
 ## TypeScript
 
 Follow [this guide](https://mdxjs.com/advanced/typescript) from the MDX docs.
+
+---
+
+# App directory
+
+## Installation
+
+For usage with the `app` directory see below.
+
+```
+npm install @next/mdx
+```
+
+or
+
+```
+yarn add @next/md
+```
+
+## Usage
+
+Create a `mdx-component.js` file in your project with the following contents:
```

Comment:
Hey there @timneutkens 

Should this file be in the app folder, or the root of the project? 

Trying to get this working on a local project and I'm guessing I won't be the only person with this question!

Reply:
Root of the project!